stellarphot/io/tess.py

```diff
@@ -225,6 +326,26 @@ def tic_id(self):
 
 @dataclass
 class TessTargetFile:
+    """ A dataclass to hold information about a TESS target file.
```

Comment:
```suggestion
    """
    A class to hold information about a TESS target file.
```
(though again, I'm not sure including class in the docstring)

Reply:
Keeping class for now.